config/checkstyle_sevntu_checks.xml

```diff
@@ -198,6 +198,9 @@
       <property name="ignoreEmptyReturns" value="false"/>
     </module>
     <module name="MoveVariableInsideIfCheck" />
+    <module name="com.github.sevntu.checkstyle.checks.coding.PreferMethodReferenceCheck">
```

Comment:
Remove this update, it shy be done after release of sevntu

Reply:
Done